contracts/test/ValidatorWalletTest.t.sol

```diff
@@ -0,0 +1,164 @@
+// Copyright 2021-2022, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity ^0.8.17;
+
+import "forge-std/Test.sol";
+import "../src/rollup/ValidatorWallet.sol";
+import "../src/rollup/ValidatorWalletCreator.sol";
+import "./RollupMock.sol";
+
+contract ValidatorWalletTest is Test {
+    // Contract instances
+    ValidatorWalletCreator creator;
+    ValidatorWallet wallet;
+    RollupMock rollupMock1;
+    RollupMock rollupMock2;
```

Comment:
I don't think we need 2 rollup, if you want to test batching you can simply call the same rollup twice.